frontend/src/containers/RecipePage/Recipe.js

```diff
@@ -3,26 +3,44 @@ import { withRouter } from 'react-router-dom';
 import Header from '../../components/Header/Header';
 import Footer from '../../components/Footer/Footer';
 import TagsBar from './TagsBar/TagsBar';
-import recipephoto from '../../assets/bread.jpg';
+import { getRecipe } from '../../lib/apiClient';
+import ScrollToTopOnMount from '../../components/ScrollToTop/ScrollToTop';
 import './Recipe.css';
 
 class Recipe extends Component {
   constructor() {
     super();
     this.state = {
-      directions: ['Mix all together', 'Bake in oven', 'Eat it all up'],
-      title: 'Easy Creamy Vegan Macaroni and Cheese',
-      snippet: 'Although this does not taste like the the traditional mac n cheese recipes most of us grew up with it will satisfy your comfort food craving while helping you avoid preservatives, dyes, meat, and dairy.',
-      ingredients: ['1 (8 ounce) package elbow macaroni', '1 cup nutritional yeast', '4 cloves garlice'],
-      difficulty: 'EASY',
-      duration: '30',
-      servings: '2',
+      directions: [],
+      title: '',
+      snippet: '',
+      ingredients: [],
+      difficulty: '',
+      duration: '',
+      servings: '',
+      photo: '',
     };
   }
 
+  componentWillMount() {
```

Comment:
According to the React documentation componentWillMount is a legacy method and should be replaced with componentDidMount.